inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/TaskConstants.java

```diff
@@ -117,6 +117,14 @@ public class TaskConstants extends CommonConstants {
     public static final String JOB_KAFKA_READ_TIMEOUT = "job.kafkaJob.read.timeout";
     public static final String TASK_KAFKA_AUTO_COMMIT_OFFSET_RESET = "task.kafkaJob.autoOffsetReset";
 
+    // Pulsar task
+    public static final String TASK_PULSAR_TENANT = "task.pulsarTask.tenant";
```

Comment:
```suggestion
    public static final String TASK_PULSAR_TENANT = "task.pulsar.tenant";
```

Reply:
The `Task` will be serialized, so this is the path name.